concrete/src/Package/Package.php

```diff
@@ -609,9 +615,13 @@ public function restore()
         return false;
     }
 
+    public function getEntityManagerProvider()
+    {
+        return new DefaultPackageProvider($this);
+    }
+
     /**
-     * Returns an array of paths directory containing package entities.
-     *
+     * @deprecated
```

Comment:
Adding info about:
* Why it's deprecated
* What to use instead
* Whether the functionality is currently preserved

to the comment makes it waaaay easier for anyone trying to update code. It can take a 2 hour upgrade job down to a few minutes if we have these everywhere.